packages/cactus-plugin-ledger-connector-fabric/README.md

```diff
@@ -1,9 +1,101 @@
 # `@hyperledger/cactus-plugin-ledger-connector-fabric`
 
-> TODO: description
+This plugin provides `Cactus` a way to interact with Fabric networks. Using this we can perform:
+* Deploy Golang chaincodes.
+* Make transactions.
+* Invoke chaincodes functions that we have deployed on the network.
+
+## Summary
+
+  - [Getting Started](#getting-started)
+  - [Usage](#usage)
+  - [Runing the tests](#running-the-tests)
+  - [Built With](#built-with)
+  - [Contributing](#contributing)
+  - [License](#license)
+  - [Acknowledgments](#acknowledgments)
+
+## Getting Started
+
+Clone the git repository on your local machine. Follow these instructions that will get you a copy of the project up and running on
+your local machine for development and testing purposes.
+
+### Prerequisites
+
+In the root of the project to install the dependencies execute the command:
+```
+npm run comfigure
+```
+
+### Compiling
+
+In the plugin root folder (/packages/cactus-plugin-ledger-connector-quorum), run this command to compile the plugin and create the dist directory:
+```
+npm run tsc
```

Comment:
@AzaharaC Does this really work in the plugin root folder instead of the project root?

Reply:
done